tutorials/kathara-external/communicating-with-the-host/README.md

```diff
@@ -0,0 +1,81 @@
+# Kathará External - Communicating with the host (Linux Only)
+
+This tutorial shows a simple example on how to allow a network scenario to directly communicate with a host interface.
+
+The same procedure can be used to connect a network scenario to a VM, a container, or another network scenario running
+on the host executing the emulation.
+
+The tutorial works only on Linux. It works even on WSL, if you install Docker directly from Linux  
```

Comment:
I would rephrase in: 
"The tutorial only works on Linux. On WSL, the tutorial works only if you install Docker directly in the Linux distribution (Docker Desktop on the Windows host is not supported)."

Reply:
Updated